models/tickets.py

```diff
@@ -73,3 +73,7 @@ def json(self):
             'notes': message_notes
         }
         # notes.json() for note in self.notes.all()]
+    
+    @classmethod
+    def find_by_tenantID(cls, tenantID):
```

Comment:
I believe this duplicates existing behavior. Could we remove this and use the inherited `find_by_id` method instead?
https://github.com/codeforpdx/dwellinglybackend/blob/development/models/base_model.py#L8